docs/templates.md

```diff
@@ -479,7 +479,9 @@ defined.
 * `.trim_end() -> String`: Removes trailing whitespace
 * `.substr(start: Integer, end: Integer) -> String`: Extract substring. The
   `start`/`end` indices should be specified in UTF-8 bytes. Negative values
-  count from the end of the string.
+  count from the end of the string. Indices are 0-based and `end` is exclusive.
+  If the `end` index is in the middle of an UTF-8 sequence, the sequence is
+  not part of the result.
```

Comment:
What if the `start` index is in the middle of a UTF-8 sequence?

btw, I *think* it's "a UTF-8 sequence", not "an ..." because it's pronounced like "you-tee-eff" (where "y" is a consonant)

Reply:
Yes, you are fully correct about the `a` / `an`.
And I checked with `start`, good point.
There it is included as soon as one byte of the codepoint is included.
I really hope for this being changed to select codepoints anways if #7953 is considered.
This feels really really strange for a substring method. :-)